src/main/java/hotelreservation/domain/HotelRepository.java

```diff
@@ -0,0 +1,9 @@
+package hotelreservation.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface HotelRepository extends JpaRepository<Hotel, String > {
+    Hotel findHotelByName(String Name);
```

Comment:
We may need to search differently but i need to think more about how. It occurs to me that hotels in different places may have the same name.

Reply:
Ya I agree this is just a place holder I think right now.